services/php-apache-nvm/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG PHP_VERSION
+FROM tugboatqa/php:${PHP_VERSION}-apache
+ENV COMPOSER_DISCARD_CHANGES=true
+ENV COMPOSER_NO_INTERACTION=true
+ENV COMPOSER_ALLOW_SUPERUSER=true
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash - \
+    && . ~/.bashrc \
+    && nvm install --lts \
+    && echo "#!/bin/bash -l" > /usr/local/bin/install-node-via-nvm.sh \
+    && echo "set -e" >> /usr/local/bin/install-node-via-nvm.sh \
+    && echo "nvm install" >> /usr/local/bin/install-node-via-nvm.sh \
+    && echo "npm install -g yarn" >> /usr/local/bin/install-node-via-nvm.sh \
+    && chmod +x /usr/local/bin/install-node-via-nvm.sh \
```

Comment:
Create a simple bash script that:
1. Is run with `bash` in login mode (hence the `-l` in the she shebang line).
2. Installs whatever version of Node is specified in `.nvmrc` within the directory that the script is called (or any of its parents).
3. Installs Yarn globally.

You might wonder, “Alfonso, you dum-dum, why not just install Yarn during image build?”, to which I would say: “Because packages installed via npm are specific to the version of Node under which they are installed, so if the consuming project installs a different version of Node.js, then they would not automatically get Yarn. Who’s the dum-dum now??”